fl4health/servers/client_level_dp_fed_avg_server.py

```diff
@@ -38,6 +40,10 @@ def __init__(
         Args:
             client_manager (ClientManager): Determines the mechanism by which clients are sampled by the server, if
                 they are to be sampled at all.
+            fl_config (Config): This should be the configuration that was used to setup the federated training.
+                In most cases it should be the "source of truth" for how FL training/evaluation should proceed. For
+                example, the config used to produce the on_fit_config_fn and on_evaluate_config_fn for the strategy.
+                NOTE: This config is DISTINCT from the Flwr server config, which is extremely minimal.
```

Comment:
Maybe put the type (ServerConfig) as well and/or some info on the distinction between fl_config and "Flwr server config" just so people know its not a substitute and stores complimentary information (if thats the case)

Reply:
Yup. Definitely a good call